pkg/util/utils.go

```diff
@@ -205,3 +205,30 @@ func MaybeChownDirRecursiveToMinikubeUser(dir string) error {
 	}
 	return nil
 }
+
+func CopyFile(src, dst string) error {
```

Comment:
Actually we have CopyFileLocal.  I'll use that.  This was just cherry-picked from an old branch